app/src/main/java/openfoodfacts/github/scrachx/openfood/features/ImagesManageActivity.kt

```diff
@@ -544,7 +544,7 @@ class ImagesManageActivity : BaseActivity() {
         }
     }
 
-    private val selectImageLauncher = registerForActivityResult(ImagesSelectActivity.Companion.SelectImageContract(binding.toolbar.title.toString()))
+    private val selectImageLauncher = registerForActivityResult(ImagesSelectActivity.Companion.SelectImageContract(""))
```

Comment:
Why? That's the title of the chooser

Reply:
registerForActivityResult() is being called before onCreate() . Getting NPE as binding is null.